configs/swarm/docker-compose.local.yml

```diff
@@ -1,46 +1,57 @@
-# This file is generated by iroha_swarm.
-# Do not edit it manually.
+# Generated by `iroha_swarm`.
+# You should not edit this manually.
+# Seed: Iroha
 
-version: '3.8'
 services:
-  irohad0:
+  builder:
+    image: hyperledger/iroha:local
     build: ../..
+    pull_policy: never
+    command: echo ok
+  irohad0:
+    depends_on:
+    - builder
+    image: hyperledger/iroha:local
+    pull_policy: never
     environment:
       CHAIN: 00000000-0000-0000-0000-000000000000
-      PUBLIC_KEY: ed012082528CCC8727333530C8F6F19F70C23882DEB1BF2BA3BE4A6654C7E8A91A7731
-      PRIVATE_KEY: 802640418A3712F4841FFE7A90B14E90BF76A6EF2A2546AC8DBBB1F442FFB8250426B082528CCC8727333530C8F6F19F70C23882DEB1BF2BA3BE4A6654C7E8A91A7731
+      PUBLIC_KEY: ed0120A98BAFB0663CE08D75EBD506FEC38A84E576A7C9B0897693ED4B04FD9EF2D18D
+      PRIVATE_KEY: 802640A4DFC16789FBF9A588525E4AC7F791AC51B12AEE8919EACC03EB2FC31D32C692A98BAFB0663CE08D75EBD506FEC38A84E576A7C9B0897693ED4B04FD9EF2D18D
       P2P_ADDRESS: 0.0.0.0:1337
       API_ADDRESS: 0.0.0.0:8080
       GENESIS_PUBLIC_KEY: ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4
+      SUMERAGI_TRUSTED_PEERS: '[{"address":"irohad2:1339","public_key":"ed01204EE2FCD53E1730AF142D1E23951198678295047F9314B4006B0CB61850B1DB10"},{"address":"irohad1:1338","public_key":"ed01209897952D14BDFAEA780087C38FF3EB800CB20B882748FC95A575ADB9CD2CB21D"},{"address":"irohad3:1340","public_key":"ed0120CACF3A84B8DC8710CE9D6B968EE95EC7EE4C93C85858F026F3B4417F569592CE"}]'
       GENESIS_SIGNED_FILE: /tmp/genesis.signed.scale
-      SUMERAGI_TRUSTED_PEERS: '[{"address":"irohad1:1338","public_key":"ed012083C85E315776FD2DDC187ECB23E608F800B313A1D614B108078EC048D5013D2D"},{"address":"irohad2:1339","public_key":"ed0120A37B7B758C952FE9429E9E35D1D71E2D8BB9364EDD077B5027ABAAC798D3230E"},{"address":"irohad3:1340","public_key":"ed0120B23E14F659B91736AAB980B6ADDCE4B1DB8A138AB0267E049C082A744471714E"}]'
     ports:
     - 1337:1337
     - 8080:8080
     volumes:
     - ./:/config
     init: true
-    command: |-
-      /bin/sh -c "
-      kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file $$GENESIS_SIGNED_FILE &&
-      irohad --submit-genesis
-      "
     healthcheck:
       test: test $(curl -s http://127.0.0.1:8080/status/blocks) -gt 0
       interval: 2s
       timeout: 1s
       retries: 30
       start_period: 4s
+    command: |-
+      /bin/sh -c "
+        kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file $$GENESIS_SIGNED_FILE &&
```

Comment:
Could we move --private-key value to env var for convince?

Reply:
Yes, but I wonder if there is a reason why it hasn't been done earlier.
@dima74 wdyt?

Reply:
It can be used to ensure that `irohad` doesn't use genesis private key. I don't mind moving it to env var

Reply:
Thank you, I'll add it to the genesis env only.

Reply:
Note that genesis private key was moved to env var in #4739 (and there are some other changes to docker-compose files)